rayon-core/Cargo.toml

```diff
@@ -21,3 +21,26 @@ libc = "0.2.16"
 lazy_static = "1"
 
 [dev-dependencies]
+
+[[test]]
+name = "stack_overflow_crash"
+path = "tests/stack_overflow_crash.rs"
+harness = false
+
+# NB: having one [[test]] manually defined means we need to declare them all
```

Comment:
Argh, that's unfortunate, though I can't immediately think of a better sol'n. We could probably do something involved where `stack_overflow_crash` invokes rustc to build the subprocess but...ugh.